src/ddspdrum/module.py

```diff
@@ -108,7 +84,7 @@ def __init__(
         assert r >= 0
         self.r = r
 
-    def __call__(self, sustain_duration=0):
+    def __call__(self, sustain_duration: float = 0):
```

Comment:
Why do we need this default? Don't we prefer that people specify this value rather than giving a weird default?

Reply:
simply put, an ADSR with a 0 sec sustain (i.e., no sustain) is an ADR envelope, which is also common.

Reply:
(especially for drums.)